src/controls/Controls.spec.js

```diff
@@ -1 +1,72 @@
-// Test away!
\ No newline at end of file
+import React from 'react';
+
+//Import renderer from react-test-renderer library
+import renderer from 'react-test-renderer';
+
+//import Dashboard component
+import Controls from './Controls';
+
+//import render from react-testing-library
+import { render, fireEvent } from '@testing-library/react';
+
+describe ('<Controls />', () => {
+    it('should match snapshot', () => {
+        const tree = renderer.create(<Controls />).toJSON();
+
+        expect(true).toMatchSnapshot();
+    });
+
+    // it('button should lock when clicked', () => {
+    //     const { getByText } = render(<Controls locked={ true } closed={ false }/>);
+
+    //     fireEvent.click(getByText(/close gate/i));
+    //     expect(getByText(/lock gate/i));
+    //     expect(getByText(/open gate/i));
+    //   });
+
+    it('close gate', () => {
+        const { getByText } = render(<Controls closed={false} locked={false}/>);
```

Comment:
this is another way to do the test, pushing the values you want and just checking the outcome without a fireEvent